swap_meet/vendor.py

```diff
@@ -1,2 +1,80 @@
 class Vendor:
-    pass
\ No newline at end of file
+        def __init__(self, inventory = None):
+            if not inventory:
+                inventory = []
+            self.inventory = inventory
+
+        def add(self, item):
+            if not item in self.inventory:
+                self.inventory.append(item)
+                return item
+
+        def remove(self, item):
+            if item in self.inventory:
+                self.inventory.remove(item)
+                return item
+            else:
+                return False
```

Comment:
A more explicit way to `return False` would be to use a guard clause instead doing it in the else block. If you use a guard clause, then you can also unindent the logic for handling a valid case where there is something to remove (lines 14 and 15).

```python
if item not in self.inventory:
     return False

self.inventory.remove(item)
     return item
```

Another approach we could take is to try to remove the item directly, and handle the ValueError that occurs if it's not there, and return False to handle it (try/except)